cucumber/authenticators_jwt/features/support/authn_jwt_helper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Utility methods for JWT authenticator
+
+module AuthnJwtHelper
+  include AuthenticatorHelpers
+
+  ACCOUNT = 'cucumber'.freeze
+  SERVICE_ID = 'raw'.freeze
+
+  def authenticate_jwt_token(service_id:, jwt_token:)
+    path = "#{conjur_hostname}/authn-jwt/#{service_id}/#{ACCOUNT}/authenticate"
+
+    payload = {}
+    payload["jwt"] = jwt_token
+
+    post(path, payload)
+  end
+
+  def create_jwt_secret(variable_name, value, service_id)
+    path = "#{ACCOUNT}:variable:conjur/authn-jwt/#{service_id}"
+    Secret.create(resource_id: "#{path}/#{variable_name}", value: value)
+  end
+
+end
+
+World(AuthnJwtHelper)
```

Comment:
What World funtion does?

Reply:
> World is just an instance of Object that you can use in every step definition, its like a library that you expose without the need to import (or require in every step definition file).

From https://stackoverflow.com/questions/4969635/what-does-the-world-method-do-in-ruby